layers/+lang/c-c++/README.org

```diff
@@ -205,10 +209,39 @@ have been disabled in favour of server,
 as recommended by =ccls= wikis.
 
 **** Debugger (dap integration)
+
+***** Adapter selection
+To select the active adapters, configure the =c-c++-dap-adapters= layer variable.
+By default only the Microsoft extension is available.
+
+| Adapter      | Supported debuggers               | DAP mode       |
+|--------------+-----------------------------------+----------------|
+| Microsoft    | gdb, lldb, Visual Studio Debugger | =dap-cpptools= |
+| CodeFreak    | gdb, lldb                         | =dap-gdb-lldb= |
+| LLVM Project | lldb                              | =dap-lldb=     |
+
+Example configuration with two adapters selected:
+#+BEGIN_SRC emacs-lisp
+  (setq-default dotspacemacs-configuration-layers
+                '((c-c++ :variables
+                         c-c++-dap-adapters '(dap-lldb dap-cpptools)))
+#+END_SRC
+
+***** Microsoft adapter installation
+To install the debug adapter you may run =M-x dap-cpptools-setup= when you are
+on Linux.
+
+***** CodeFreak adapter installation
 To install the debug adapter you may run =M-x dap-gdb-lldb-setup= when you are
 on Linux or download it manually from [[https://marketplace.visualstudio.com/items?itemName=webfreak.debug][Native Debug]] and adjust the
 =dap-gdb-lldb-path= variable.
 
+***** LLDB adapter installation
+Install the =lldb-vscode= program. It is a part of the =lldb= apt package.
```

Comment:
This only applies to Ubuntu or any distro that uses `apt`.
A more generic description could be:
```text
Install the =lldb-vscode= program. It usually comes with `lldb` from your distribution's package repository.
Alternatively, ....
```

Reply:
Done. Thanks!